content_handler/runtime_holder.cc

```diff
@@ -329,11 +339,11 @@ void RuntimeHolder::InitRootBundle(std::vector<char> bundle) {
       GetUnzipperProviderForRootBundle());
 }
 
-void RuntimeHolder::HandleAssetPlatformMessage(
+bool RuntimeHolder::HandleAssetPlatformMessage(
     ftl::RefPtr<blink::PlatformMessage> message) {
   ftl::RefPtr<blink::PlatformMessageResponse> response = message->response();
   if (!response)
-    return;
+    return true;
```

Comment:
`return false`? We still want to flow down the path where `HandlePlatformMessage` completes an empty response.